src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
Yup, exactly what I was thinking. The argument needs to be changed in the `context-pure` mixin. Could you submit a PR for this?